modules/core/shared/src/main/scala/coursier/maven/PomParser.scala

```diff
@@ -95,7 +95,12 @@ object PomParser {
 
     var description = ""
     var url = ""
-    val licenses = Nil // TODO
+    var licenses = Nil // Left in for binary compat
```

Comment:
So it was complaining about this as well :/ Is it true that since this was inferred previously as a `scala.collection.immutable.Nil.type` it now breaks binary compatibility if `PomParser` doesn't have a licenses filed that returns a `Nil.type`?

Reply:
Oh, I wanted to mention it too: this class is private, so you can just add the filter printed by `mimaReportBinaryIssues` in [`Mima.scala`](https://github.com/coursier/coursier/blob/85df5e041156cb89105ab3ac8d89f098adce1daa/project/Mima.scala#L74).

Reply:
Ah, ok cool. Added.